docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,282 @@
+```
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import get_scale_factor_vectors
+
+scale_factors = get_scale_factor_vectors(circuit, degree=2, fold_multiplier=2)
+
+scale_factors
+```
+
+
+In the noise scaled circuits created using the above scale factor vectors:
+
+- The term $1$ in the monomial terms basis corresponds to the `degree = 0` term in the polynomial which is equivalent to
+the $\lambda_1^0\lambda_2^0\lambda_3^0\lambda_4^0$ term. Due to this term, the first noise-scaled circuit is unchanged.
+
+- due to the $λ_1$ term in the monomial basis, the second noise-scaled circuit only scales the first layer in the circuit.
+
+- due to the $λ_2$ term in the monomial basis, the next noise-scaled circuit only scales the second layer in the circuit
+
+- and so on.
+
+```{code-cell} ipython3
+print(f"Total number of noise scaled circuits created: {len(scale_factors)}")
+```
+
+As the `fold_multiplier` is changed, the number of scaled circuits remains the same but how the layers are scaled
+is altered.
+
+```{code-cell} ipython3
+
+
+scale_factors_diff_fold_multiplier = get_scale_factor_vectors(
+   circuit,
+   degree=2,
+   fold_multiplier=3)
+
+
+print(f"Total number of noise-scaled circuits created with different" 
+      f" fold_multiplier: {len(scale_factors_diff_fold_multiplier)}")
+
+print(f"Scale factor for some noise scaled circuit with degree=2 "
+      f"and fold_multiplier=2: \n {scale_factors[-2]}")
+
+print(f"Scale factor for some noise scaled circuit with degree= 2 "
+     f"but fold_multiplier=3:  \n {scale_factors_diff_fold_multiplier[-2]}")
+```
+
+
+Both the number of noise scaled circuits and scale factor vectors are changed when a different value for `degree` is used while keeping everything else the same.
+
+
+```{code-cell} ipython3
+
+
+scale_factors_diff_degree = get_scale_factor_vectors(
+   circuit,
+   degree=3,
+   fold_multiplier=2)
+
+print(f"Total number of noise scaled circuits created: "
+      f"{len(scale_factors_diff_degree)}")
+```
+
+
+Thus, even though `degree` and `fold_multiplier` are required to use {func}`.execute_with_lre`, they function as a tunable
+hyperparameter affecting the performance of the technique.
+
+
+## Chunking a circuit into fewer layers
+
+
+When you have a large circuit, the size of the sample matrix increases as the number of monomial terms scale polynomially. The size of the sample matrix influences sampling cost. In such a case, a circuit of 100 layers could be grouped into 4 chunks where each chunk consists of multiple collated layers. The noise scaling function {func}`.multivariate_layer_scaling`
+treats each chunk as a layer to be scaled when the parameter `num_chunks` is used. Thus, for the 100 layer circuit grouped into 4 chunks with `degree = 2` and `fold_multiplier = 2`, only 15 noise-scaled circuits are created i.e. sample matrix is reduced to dimension $15 \times 15$.
```

Comment:
```suggestion
treats each chunk as a layer to be scaled when the parameter `num_chunks` is used. Thus, for the 100 layer circuit grouped into 4 chunks with `degree = 2` and `fold_multiplier = 2`, only 15 noise-scaled circuits are created i.e. sample matrix is reduced to dimension $15 \times 15$ instead of $x \times x$.
```

Where `x` is the number of circuits without chunking.